src/components/EventsTimeline/stories/basic.story.mdx

```diff
@@ -0,0 +1,295 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
+import { EventsTimeline } from '../EventsTimeline';
+import { EventsTimelineTooltip } from './EventsTimelineTooltip';
+import {
+  events,
+  start,
+  end,
+  decorators,
+  toTimelines,
+  onChange,
+  onHide,
+  onZoom,
+  onZoomStart,
+  onZoomEnd,
+  dateFormatter,
+  dateFormatterCode,
+} from './helper';
+
+<Meta title="Events/EventsTimeline" decorators={decorators} />
+
+# Events Timeline
+
+`EventsTimeline` representing events as blocks on timeline between `start` and `end` date.
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+### Basic usage:
+
+Imports and props:
+
+```typescript
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import {
+  EventsTimeline,
+  TimelineEvent,
+  EventTimelineView,
+  EventTimelineType
+} from '@cognite/gearbox';
+
+const events: TimelineEvent[] = [
+  {
+    id: 0,
+    view: EventTimelineView.fill,
+    type: EventTimelineType.continuous,
+  },
+  {
+   id: 1,
+    view: EventTimelineView.outline,
+    type: EventTimelineType.continuous,
+  },
+  {
+   id: 2,
+    view: EventTimelineView.outline,
+    type: EventTimelineType.discrete,
+  },
+];
+const end = Date.now();
+const start = end - 60 * 60 * 1000;
+```
+<Preview>
+  <Story name="Basic Usage" >
+    <EventsTimeline  events={events} start={start} end={end} />
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={EventsTimeline} />
+
+### Types
+
+`TimelineEvent`, `TimelineRuler`, `TimelineZoom`, `TimelineSize`, `CogniteEventForTimeline`
+type can be imported from `@cognite/gerabox`:
+
+```typescript
+import {
+  CogniteEventForTimeline,
+  TimelineEvent,
+  TimelineRuler,
+  TimelineZoom,
+  TimelineSize,
+} from '@cognite/sdk';
+```
+
+## Examples
+
+### Split to timelines
+
+```typescript
+const toTimelines = ({ id }: CogniteEventForTimeline) => {
+  return !id ? '#3b7c14' : id === 1 ? '#ee5d7d' : '#cccccc';
+};
+```
+
+<Preview>
+  <Story name="Split to timelines">
+    <EventsTimeline
+      events={events}
+      start={start}
+      end={end}
```

Comment:
I think you should be able to shift it so it looks better in storybook. (now the indentation is wrong). what do you think?

Reply:
it is something that happens across all storybook and authors work on it. I'll try to upgrade storybook to the latest version (v5.3.7) after merging this PR, it might fix this issue

Reply:
if you choose to indent it, there are other places as well